### PR TITLE
Add FEI

### PR DESCRIPTION
**Pull request by Joeysantoro**

### CODE REVIEW

File: markets/aave/commons.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ const MOCK_CHAINLINK_AGGREGATORS_PRICES = {
   BUSD: oneEther.multipliedBy('0.00736484').toFixed(),
   DAI: oneEther.multipliedBy('0.00369068412860').toFixed(),
   ENJ: oneEther.multipliedBy('0.00029560').toFixed(),
+  FEI: oneEther.multipliedBy('0.00042414').toFixed(),
```
Comment (miguelmtzinf) [inline markets/aave/commons.ts:11]: Please, move this line before the USD mock price please, just to make it easier to follow the evolution of new listings
https://github.com/aave/protocol-v2/pull/200/files#diff-fdb3544e2ccf0395080fe6b84d8bc923ebc899a2b5ac4fb70235d85983ed43c1R28

**Comment by miguelmtzinf** [comment]
Please, add the following two lines at line 170 (before the USD line) to make the tests work:

```
      xSUSHI: mockTokens.xSUSHI.address,
      FEI: mockTokens.FEI.address,
```
You can try yourself the test running `npm run test`.

Thanks for your contribution!

File: markets/aave/commons.ts (hunk 1 above)
Comment (Joeysantoro) [reply markets/aave/commons.ts:11]: They appear to be alphabetical which is why I put it here. Happy to move it if thats not intentional

**Comment by Joeysantoro** [comment]
>       xSUSHI: mockTokens.xSUSHI.address,
>       FEI: mockTokens.FEI.address,

Added, and verified the tests pass (two failed with what appeared to be rounding errors)


```
  1) LendingPool: Borrow/repay (stable rate)
       User 0 deposits 1000 DAI, user 1 deposits 2 WETH and borrow 100 DAI at stable rate first, then 100 DAI at variable rate, repays everything. User 0 withdraws:

      AssertionError: expected '5014915580971766967821' to be almost equal or equal '5014915580967737387859' for property totalLiquidity
      + expected - actual

      -5014915580971766967821
      +5014915580967737387859

      at expectEqual (test/helpers/actions.ts:323:30)
      at Object.exports.deposit (test/helpers/actions.ts:90:9)
      at process._tickCallback (internal/process/next_tick.js:68:7)

  2) LendingPool: Borrow/repay (variable rate)
       User 0 deposits 1000 DAI, user 6 deposits 2 WETH and borrow 100 DAI at variable rate first, then 100 DAI at stable rate, repays everything. User 0 withdraws:

      AssertionError: expected '6005911616672537885645' to be almost equal or equal '6005911616669322773812' for property totalLiquidity
      + expected - actual

      -6005911616672537885645
      +6005911616669322773812

      at expectEqual (test/helpers/actions.ts:323:30)
      at Object.exports.deposit (test/helpers/actions.ts:90:9)
      at process._tickCallback (internal/process/next_tick.js:68:7)
      ```

File: markets/aave/commons.ts (hunk 1 above)
Comment (miguelmtzinf) [reply markets/aave/commons.ts:11]: Whoops, you are right, thanks for the catch!

**Pull request closed without merging**